GW2EIEvtcParser/EncounterLogic/Raids/W8/GreerTheBlightbringer.cs

```diff
@@ -137,4 +234,238 @@ internal override void ComputePlayerCombatReplayActors(PlayerActor player, Parse
         }
 
     }
+
+    private static void AddScatteringSporeblast(NPC target, ParsedEvtcLog log, CombatReplay replay)
+    {
+        // Scattering Sporeblast - Indicator
+        if (log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.GreerScatteringSporeblastIndicator, out var sporeblasts))
+        {
+            foreach (EffectEvent effect in sporeblasts)
+            {
+                (long start, long end) lifespan = effect.ComputeLifespan(log, effect.Duration);
+                var indicator = new CircleDecoration(100, lifespan, Colors.LightOrange, 0.2, new PositionConnector(effect.Position));
+                replay.Decorations.Add(indicator);
+            }
+        }
+    }
+
+    private static void AddSweepTheMoldRakeTheRot(NPC target, ParsedEvtcLog log, CombatReplay replay, uint radius)
+    {
+        // Swepp the Mold / Rake the Rot - Indicator
+        if (log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.GreerSweepTheMoldRakeTheRotIndicator, out var indicators))
+        {
+            foreach (EffectEvent effect in indicators)
+            {
+                (long start, long end) lifespan = effect.ComputeLifespan(log, effect.Duration);
+                var cone = new PieDecoration(radius, 120, lifespan, Colors.LightOrange, 0.2, new PositionConnector(effect.Position)).UsingRotationConnector(new AngleConnector(effect.Rotation.Z + 90));
+                replay.Decorations.Add(cone);
+            }
+        }
+    }
+
+    private static void AddEnfeeblingMiasma(NPC target, ParsedEvtcLog log, CombatReplay replay)
+    {
+        // Enfeebling Miasma - Cone Indicator
+        if (log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.GreerEnfeeblingMiasma, out var miasmaIndicator))
+        {
+            foreach (EffectEvent effect in miasmaIndicator)
+            {
+                (long start, long end) lifespan = effect.ComputeLifespan(log, 6000);
+                var cone = new PieDecoration(2000, 60, lifespan, Colors.LightOrange, 0.2, new PositionConnector(effect.Position)).UsingRotationConnector(new AngleConnector(effect.Rotation.Z + 90));
+                replay.Decorations.Add(cone);
+            }
+        }
+
+        // Enfeebling Miasma - Gas Circles
+        if (log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.GreerEnfeeblingMiasmaGasMoving, out var miasmaAnimation))
```

Comment:
&& log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.GreerEnfeeblingMiasmaGasClouds, out var miasmaClouds), no need to do a fetch for each "animation"